src/main/java/com/cedrus/patpetillo/springkafkapingpong/model/ServeBallResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.cedrus.patpetillo.springkafkapingpong.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class ServeBallResponse {
+    @JsonProperty
+    private ApiResponse apiResponse;
+
+    public ServeBallResponse(boolean successInd) {
+        if (successInd) {
+            this.apiResponse = new ApiResponse();
+            this.apiResponse.setCode("OK: 200");
```

Comment:
Generally think of return codes as something that will be system handled. Something like OK should go in a message, not a code field. This kind of format leads to logical breaks due to spaces, capitalization, etc.